src/Verify/VirtualizedRunHelper.cs

```diff
@@ -12,82 +14,77 @@ class VirtualizedRunHelper
     };
 
     public VirtualizedRunHelper(Assembly userAssembly)
+        : this(GetOriginalCodeBaseRoot(userAssembly),
+            Environment.CurrentDirectory,
+            Path.DirectorySeparatorChar,
+            path => File.Exists(path) || Directory.Exists(path))
+    {
+    }
+
+    private static string GetOriginalCodeBaseRoot(Assembly userAssembly) =>
+        AttributeReader.TryGetSolutionDirectory(userAssembly, false, out var solutionDir)
+            ? solutionDir : AttributeReader.GetProjectDirectory(userAssembly);
+
+    internal VirtualizedRunHelper(string originalCodeBaseRoot, string currentDir, char runtimeSeparator, Func<string, bool> pathExists)
```

Comment:
`runtimeSeparator` is a weird naming (not clear what it is by just looking on the name) - I'd keep it `directorySeparatorChar`